docs/explanation/remote-build.rst

```diff
@@ -8,19 +8,18 @@ different architectures.
 Architectures supported by Launchpad can be found
 :ref:`here<supported-architectures>`.
 
-Open vs closed source
----------------------
+Public vs. Private projects
+---------------------------
 
-By default, prospective snaps must be open source because the build will be
-publicly available.
+By default, prospective snaps will be publicly uploaded to `Launchpad`_.
```

Comment:
```suggestion
By default, prospective snaps are publicly uploaded to `Launchpad`_.
```

When describing a notional time and place that isn't immediate, which is when references and explanations typically operate, it's usually simpler to use present tense.